src/services/middlewareServices.js

```diff
@@ -48,7 +49,8 @@ const PROVISION_SERVICES = [
   DEFAULT_SERVICES.APICURIO,
   DEFAULT_SERVICES.THREESCALE,
   DEFAULT_SERVICES.FUSE_MANAGED,
-  DEFAULT_SERVICES.RHSSO
+  DEFAULT_SERVICES.RHSSO,
+  DEFAULT_SERVICES.MDC
```

Comment:
Will this not result in the webapp trying to provision the MDC every time, regardless of whether it exists or not? From what I understood the MDC is an optional add-on in the current installer and may not be found. Is that still correct?

Reply:
@aidenkeating I have very limited knowledge of the webapp. 
Our target is only to *get url of the MDC* for our walkthorugh. 
MDC will be always installed and do not need to be proisioned. 
If you know any secret on how we can get access to MDC url god to bless you. What we have is MDC server running in mdc namespace with some route.

I see URL available in Application manifest (see above) in MDC so Web app already knows this url, but I personally struggle to get easy way to expose this url to walkthough. Maybe we can have something custom for MDC that will be able to consume value above?


@wei-lee correct me if I'm wrong.

Reply:
mdc will be installed by default now (once we merge the feature branch)

Reply:
I am making it optional and will update this pr with my commit as soon as I am done with it